github_repos/LIQUID_HIVE_25/mock_backend.py

```
@@ -0,0 +1,77 @@
+    
+    return {"answer": response}
+
+@app.post("/auth/token")
+async def create_token(request: TokenRequest):
+    # Mock token generation
+    token = f"mock_token_{request.user_id}_{int(time.time())}"
+    return {"token": token}
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Ruff (0.13.3)

Possible binding to all interfaces

(S104)